kolibri/plugins/coach/assets/src/views/reports/ReportsLearnerReportPage.vue

```diff
@@ -4,99 +4,101 @@
     <KPageContainer>
       <ReportsLearnerHeader />
     </KPageContainer>
-
-    <KPageContainer>
-      <HeaderTabs>
-        <KTabsList
-          ref="tabList"
-          :tabsId="REPORTS_LEARNERS_TABS_ID"
-          :ariaLabel="$tr('reportLearners')"
-          :activeTabId="ReportsLearnersTabs.REPORTS"
-          :tabs="tabs"
-          @click="() => saveTabsClick(REPORTS_LEARNERS_TABS_ID)"
-        />
-      </HeaderTabs>
-      <KTabsPanel
-        :tabsId="REPORTS_LEARNERS_TABS_ID"
-        :activeTabId="ReportsLearnersTabs.REPORTS"
+    <KGrid >
+      <KGridItem
+        :layout12="{ span: 6 }"
+        :layout8="{ span: 4 }"
       >
-        <KGrid>
-          <KGridItem :layout12="{ span: $isPrint ? 12 : 6 }">
-            <h2>{{ coachString('lessonsAssignedLabel') }}</h2>
-            <CoreTable :emptyMessage="coachString('lessonListEmptyState')">
-              <template #headers>
-                <th>{{ coachString('titleLabel') }}</th>
-                <th>{{ coreString('progressLabel') }}</th>
-              </template>
-              <template #tbody>
-                <transition-group
-                  tag="tbody"
-                  name="list"
+        <KPageContainer class="content-spacing left-container">
```

Comment:
`left-container` and `right-container` are the same. I think we can consolidate it into 1 class?